src/bridge/test-websocketstream.c

```diff
@@ -178,6 +178,32 @@ test_basic (TestCase *test,
   g_object_unref (channel);
 }
 
+static void
+test_bad_port (TestCase *test,
+               gconstpointer data)
+{
+  test->port = 1023;
```

Comment:
That's not any magic number, right? If there happens to be something listening on that port, this test would fail? Could it be port 65537 or so, which is truly invalid? (and then it's probably something else than `not-found`)

Reply:
Correct.

I could try your suggestion, but I guess we'd rather get a rejection saying that the json options object contains an invalid port number....

Do you have a better idea for what we can do here?  I didn't want to use a strange IP address because that is somewhat likely to actually send packets out in search of that host.  Ditto invalid hostnames (DNS lookups).  Maybe we could use an invalid multicast address?